macros/sfdc_formula_view_fields.sql

```diff
@@ -1,7 +1,8 @@
-{%- macro sfdc_formula_view_fields(join_to_table, source_name = 'salesforce') -%}
+{%- macro sfdc_formula_view_fields(join_to_table, source_name = 'salesforce', inclusion_fields=none) -%}
 
+{%- set not_null_value = true -%}
```

Comment:
why should this be harcoded as true here?

Reply:
Same as above